javascript.js

```diff
@@ -0,0 +1,15 @@
+const ham = document.querySelector('.button');
+const navbar = document.querySelector('.nav-bar');
+const clos = document.querySelector('.clos');
```

Comment:
- [ ] It would be better if you name your variables a name that everyone can understand so when you are doing pair programming your  partner will understand what is it just based on the name so please try to fix that